README.md

```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
   
 </p>
 
-# Try out Opta
+# Features
+
+### Cloud services
+Opta supports the 3 major clouds - AWS, GCP and Azure. It has modules for the most commonly used services in these clouds like:
+* Microservices (powered by [Kubernetes](https://docs.opta.dev/architecture/aws/))
+* Databases - Postgres, MySQL, Redis
+* Serverless workloads
+* Networking - VPCs, Subnets, Load balancers
+* Cloudfront
```

Comment:
this list of features is not cloud specific so we could say Content Delivery Network (CDN)